tonic/src/transport/channel/tls.rs

```diff
@@ -51,6 +51,13 @@ impl ClientTlsConfig {
         ClientTlsConfig { certs, ..self }
     }
 
+    /// Sets the multiple CA Certificates against which to verify the server's TLS certificate.
+    pub fn ca_certificates(self, ca_certificates: Vec<Certificate>) -> Self {
```

Comment:
Let's take an `impl IntoIterator<Item = Certificate>` here, instead?

Reply:
Sounds good to me. Updated to use it.